docs/fsharp/language-reference/compiler-messages/new-message

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env bash
```

Comment:
what is this file? you should only have md files here

Reply:
It's a helper script for anyone else that wants to create a new file easily. It generates the metadata header for a new file given the error message and the title you want to assign. I found it very helpful when creating new entries, especially given that there are over a thousand to work through!  Does having non-md files in here break workflows? Are there more appropriate places for public helper scripts to live?

Reply:
It doesn't seem that this page is being published so I guess that's ok. But unusual for us to have this along with the documentation files.